// This is a bug in the reference solution.

procedure foo(var integer x) {
    x = x + 1;
}

procedure main() returns integer {
    const tuple(integer x, integer y) x = (1, 3);
    call foo(x.y);
    x.y -> std_output;
    return 0;
}
//CHECK:TypeError on Line 9:
